webapp/channels/src/components/__snapshots__/textbox.test.tsx.snap

```diff
@@ -2,97 +2,8 @@
 
 exports[`components/TextBox should match snapshot with additional, optional props 1`] = `
 <div
-  className="textarea-wrapper textarea-wrapper--preview"
+  className="textarea-wrapper"
 >
-  <Connect(SuggestionBox)
```

Comment:
Oh yeah, that's a change I forgot to mention. Currently, the `SuggestionBox` is still visible when the preview is rendered and its hidden by CSS. That caused me some issues so I made it so that only one or the other of them is rendered at a time